UPGRADE-3.1.md

```diff
@@ -107,7 +107,7 @@ FrameworkBundle
    framework:
        cache:
            pools:
-               serializer:
+               cache.serializer:
```

Comment:
I think I prefer to be explicit. Whenever possible we try to use the full service name in configuration and that's what is done here. If the end user does not want to prefix their service name with `cache.`, it's not a problem.